joss_paper/paper.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Zarrtraj: A Python package for streaming molecular dynamics trajectories from cloud services'
+tags:
+  - streaming
+  - molecular-dynamics
+  - file-format
+  - mdanalysis
+  - zarr
+authors:
+  - name: Lawson Woods
+    orcid: 0009-0003-0713-4167
+    affiliation: 1 
+  - name: Hugo Macdermott-Opeskin
+    orcid: 0000-0002-7393-7457
+    affiliation: 1
+  - name: Oliver Beckstein
+    orcid: 000-0003-1340-0831
+    affiliation: 1
+  - name: Edis Jakupovic 
+    affiliation: 1
+  - name: Yuxuan Zhuang
+    orcid: 0000-0003-4390-8556
+    affiliations: 1
+  - name: Richard J Gowers
+    orcid: 0000-0002-3241-1846
+    affiliations: 1
+affiliations:
+ - name: Placeholder
+   index: 1
+date: 22 September 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Molecular dynamics simulations provide a microscope into the behavior of 
+atomic-scale environments otherwise prohibitively difficult to observe, however,
+the resulting trajectory data is too often siloed in a single institutions' 
+HPC environment, rendering it unusable by the broader scientific community.
+Zarrtraj enables these trajectories to be read directly from cloud storage providers
+like AWS, Google Cloud, and Microsoft Azure into MDAnalysis, a popular Python 
+package for analyzing trajectory data, providing a method to open up access to
```

Comment:
Make clear why anyone would actually want to access MD trajectories --- what's the motivation for streaming?

I'd also cite efforts like MDDB and the eLife MDverse paper https://elifesciences.org/articles/90061

Reply:
And machine learning, of course -- @hmacdope may have suitable citations at hand.